### DISCUSSION TITLE
how to attach to remote process

### DISCUSSION THREAD

**Discussion by jeusdi** [mfussenegger/nvim-dap#1125] [Q&A]
Context: Java, Debugging attaching to remote proces/port.

By default, I see lazyvim starts up a new instance of my application when I'm trying to debug my spring boot application.

[Here you can see this behavior](https://asciinema.org/a/caGEJRF5pKEVkAz4YdEe9iMss
).

I need to attach my debug session to a remote process listening on port 5005.

I don't quite figure out how to attach it instead of start a new process.

Any ideas?

**Comment by mfussenegger** [comment]
See https://github.com/mfussenegger/nvim-dap/wiki/Java#configuration

**Comment by LenoCode** [comment]
One question
I am using dap to debug my neovim plugin.
I was able to configure it and it works. I run the instance, and from another nvim I attach to the process. My problem is that when I write some values in expression nothing gets evaluated . Is that normal behavior when you attach to the different process
